### DISCUSSION TITLE
PruneStale and AutoClossing Questions

### DISCUSSION THREAD

**Discussion by dkr91** [renovatebot/renovate#27521] [Request Help] [answered]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us what version of Renovate you run.

37.173.0

### If you're self-hosting Renovate, select which platform you are using.

GitLab self-hosted

### What is your question?

Hey, we have issues with Renovate Autoclosing MRs and then reopening them with the same branch creating a lot of unwanted CI-Jobs in the Process.
Does disabling the [pruneStaleBranches ](https://docs.renovatebot.com/configuration-options/#prunestalebranches) option prevent this from happening? And is there any way to have orphan branches delted without autoclossing PRs?

### Logs (if relevant)

<details><summary>Logs</summary>

```

Copy/paste the relevant log(s) here, between the starting and ending backticks

```

</details>

**Comment by rarkins** [comment, marked as answer]
Renovate will autoclose any of its existing PRs which it thinks shouldn't exist. If you are seeing autoclosing which you think is mistaken, then you/we should be looking into that, and not disabling autoclosing. e.g. is the autoclosing happening because Renovate is unable to reach certain registries temporarily?

**Reply by dkr91** [reply]
Sorry for answering late. We now know that this happens when we update our Local Artifactory. My question is if there is any way to prevent that from happening when we know that it will be unreachable or is the best solution to shutdown Renovate during the update to prevent Renovate from Autoclosing the MRs?

**Reply by rarkins** [reply]
Yes, https://docs.renovatebot.com/configuration-options/#abortonerror should work for this
